ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/ParseReportingUrl.java

```diff
@@ -276,6 +353,13 @@ private List<String[]> readPairsFromFile(String fileLocation, String paramName)
     }
   }
 
+  private String extractReportingUrl(ObjectNode payload) {
+    if (!payload.path(Attribute.REPORTING_URL).isMissingNode()) {
+      return payload.path(Attribute.REPORTING_URL).asText();
+    }
+    return payload.path("metrics").path("url2").path("top_sites_contile_reporting_url").asText();
```

Comment:
```suggestion
    return payload.path("metrics").path("url").path("top_sites_contile_reporting_url").asText();
```

This should be `url` at this point in the pipeline. It gets translated to url2 at the BigQuery write step.